isis/src/apollo/apps/apollopanstitcher/main.cpp

```diff
@@ -166,7 +166,7 @@ void IsisMain() {
 
   //parameters for maximum correlation autoregestration
   // see: ///usgs/pkgs/isis3nightly2011-09-21/isis/doc/documents/patternSMatch/patternSMatch.html#DistanceTolerance
-  FileName fiducialPvl("$apollo15/templates/apolloPanFiducialFinder.pvl");
+  FileName fiducialPvl("$ISISROOT/appdata/templates/apollo/PanFiducialFinder.def");
```

Comment:
Is this file supposed to be named apolloPanFiducialFinder.def?

Reply:
Stuart brought up the point that since it will be under a templates/apollo directory, there is no reason to tack on apollo in front of the file name.